tests/retribert/test_tokenization_retribert.py

```diff
@@ -0,0 +1,340 @@
+# coding=utf-8
+# Copyright 2022 The HuggingFace Team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" Testing suite for the RetriBERT tokenizer. """
+
+
+import os
+import unittest
+
+from transformers import RetriBertTokenizer, RetriBertTokenizerFast
+from transformers.models.bert.tokenization_bert import (
+    VOCAB_FILES_NAMES,
+    BasicTokenizer,
+    WordpieceTokenizer,
+    _is_control,
+    _is_punctuation,
+    _is_whitespace,
+)
+from transformers.testing_utils import require_tokenizers, slow
+
+from ..test_tokenization_common import TokenizerTesterMixin, filter_non_english
+
+
+@require_tokenizers
+class RetriBertTokenizationTest(TokenizerTesterMixin, unittest.TestCase):
+    """This is a duplicate of BertTokenizationTest, since RetriBertTokenizer is identical to BertTokenizer."""
```

Comment:
I just suggest this small change so that the syntax used is similar to that used in other places in the codebase like:
https://github.com/huggingface/transformers/blob/da47c264f9a881f5db5f6fbb59a30c95e428571f/src/transformers/models/hubert/modeling_hubert.py#L74

```suggestion

# Copied from transformers.tests.bert.test_modeling_bert.py with Bert->RetriBert
@require_tokenizers
class RetriBertTokenizationTest(TokenizerTesterMixin, unittest.TestCase):
```

Reply:
Hi @SaulLu, thanks for your reply! These changes make sense to me, I will adopt them.